app/src/main/java/protect/card_locker/MainActivity.java

```diff
@@ -162,6 +163,22 @@ public boolean onActionItemClicked(ActionMode inputMode, MenuItem inputItem) {
 
                 return true;
             }
+            else if(inputItem.getItemId() == R.id.action_archive_unarchive){
+                for (LoyaltyCard loyaltyCard : mAdapter.getSelectedItems()) {
+                    Log.e(TAG, "Archiving card: " + loyaltyCard.id);
+                    int num;
+                    if(loyaltyCard.archiveStatus == 0){
+                        num = 1;
+                    }
+                    else{
+                        num = 0;
+                    }
```

Comment:
I think this will be very confusing for people, they select multiple cards and instead of them all archiving or unarchiving they'll flip state around.